cpg-language-python/src/test/resources/python/with.py

```diff
@@ -0,0 +1,5 @@
+with open("file.txt", "r") as file:
```

Comment:
I would prefer a test case using a class that implements the context manager protocol, similar to what we sketched together

Reply:
Yes, but please also keep this test, too. It's a very common pattern in python code.

Reply:
Done.